Adafruit_FXAS21002C.cpp

```diff
@@ -263,3 +281,97 @@ void Adafruit_FXAS21002C::standby(boolean standby) {
     active_bit.write(0x03);
   }
 }
+
+/**************************************************************************/
+/*!
+    @brief  Configures the device with certain output data rate(ODR)
+            Supports ODRs: 8000Hz, 400Hz, 200Hz, 100Hz, 50Hz, 25Hz, 12.5Hz
+    @param   ODR : the output data rate to be set to the gyroscope
+*/
+/**************************************************************************/
+void Adafruit_FXAS21002C::setODR(float ODR) {
+  Adafruit_BusIO_Register CTRL_REG1(i2c_dev, GYRO_REGISTER_CTRL_REG1);
+  Adafruit_BusIO_RegisterBits datarate_bits(&CTRL_REG1, 3, 2);
+
+  /* CTRL_REG1 should only be set in Standby or Ready mode. First enter Standby
+   * mode */
+  standby(true);
+
+  if (ODR == GYRO_ODR_800HZ) {
+    datarate_bits.write(0b000);
+    _ODR = ODR;
```

Comment:
Should work to just have
```
_ODS = ODR;
```
once right after this big if/else if conditional block. That way don't need to set it inside each conditional like this.

Reply:
Yes.  `_ODS = ODR;` is under each conditional to make sure _ODR is only updated if the input ODR is valid (now ODR is a float instead of an enum. So ODR could be invalid values). 
We can move the _ODR update outside the conditional block. Just that the user needs to give setODR(float ODR) a valid ODR